src/strings/index.tsx

```diff
@@ -96,10 +96,10 @@ const strings = new LocalizedStrings({
     PURPOSE: 'Purpose',
     TARGET_RH: 'Target %RH',
     SEED_TYPE: 'Seed Type',
-    NUMBER_OF_SEEDS_FILLED: 'Number of Seeds Filled',
+    NUMBER_OF_SEEDS_FILLED: '# Seeds Filled',
```

Comment:
they wanted '#' instead of 'Number of'?

Reply:
"Number of" was taking two lines, and all designs have #